### ISSUE TITLE
files open with this weird red highlighting, if in normal mode the cursor visits the line the highlighting goes away ?

### ISSUE THREAD

**Issue by a-eid** [asvetliakov/vscode-neovim#86]
<img width="399" alt="Screen Shot 2019-12-10 at 16 18 00" src="https://user-images.githubusercontent.com/19273413/70537530-40bf0d00-1b69-11ea-9c9d-584315bd10a2.png">
<img width="1022" alt="Screen Shot 2019-12-10 at 16 20 55" src="https://user-images.githubusercontent.com/19273413/70537531-4157a380-1b69-11ea-88e8-f303cd0860c0.png">

**Comment by a-eid** [comment]
going throw the `README` disabling `editor.scrollBeyondLastLine` does solve the issue.

**Comment by asvetliakov** [comment]
@a-eid i don't think ```editor.scrollBeyondLastLine``` should cause this... do you have any vim plugins/options enabled ?

**Comment by a-eid** [comment]
<img width="718" alt="Screen Shot 2019-12-10 at 16 28 59" src="https://user-images.githubusercontent.com/19273413/70538078-2d607180-1b6a-11ea-9009-7d1ee98b6af0.png">

**Comment by a-eid** [comment]
I think after a min or 2 it does go away for some reason !

**Comment by asvetliakov** [comment]
what is your value for ```set spell=``` ? can you try with plugins and other conf (if exists) disabled? (half of them need to be disabled for vscode anyway)

**Comment by a-eid** [comment]
<img width="394" alt="Screen Shot 2019-12-10 at 16 54 35" src="https://user-images.githubusercontent.com/19273413/70540344-c0e77180-1b6d-11ea-8b8d-5931d937db2f.png">

disabled all plugins and issue still persists.

**Comment by asvetliakov** [comment]
try to disable spelling

**Comment by a-eid** [comment]
that might have been it.
thank you @asvetliakov

**Comment by asvetliakov** [comment]
Well it's good news at least that spelling works, so people have a choice either to use vscode spell plugin or vim one 😄

**Comment by David-Else** [comment]
@asvetliakov I am trying to get the built in neovim spelling to work without luck. I made a `init.vim` in `$HOME/.config/nvim` and added:
```
set spell
set spelllang=en
```
I configured the extension to point to it:
```
"vscode-neovim.neovimInitPath": "/home/david/.config/nvim"
```
and I am not seeing any red squiggles to indicate it is working. I also tried the command `set spell` directly in the command input. It works in actual neo vim on my machine (Linux Centos 8.1)

Would be great to be able to toggle it on/off too :) Cheers!
